src/main/resources/scripts/V1.0__modify_dummy_data_table.sql

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
nitpick: rename these script files after the content, so this could be `query_by_type.sql`

same for the scripts made from the INSERTs/UPDATEs, ex. insert_dummy_data.sql